# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/MediaEngineWebRTCAudio.cpp
# Commit: b004b579d1ce
# Full Hash: b004b579d1ce8fa91acc2330c436340b099ceccd
# Author: Chun-Min Chang <chun.m.chang@gmail.com>
# Date: 2022-01-28 04:41:16
# Regressor Bug: 1742655
# File Overlap Count: 2
# Description:
#   Bug 1742655 - Rename functions r=pehrsons
#   
#   Rename {Close, Open}AudioInput to {Disc, C}onnectDeviceInput
#   
#   Depends on D134533
# ==============================================================================

diff -r 8cb7b8231c98 -r b004b579d1ce dom/media/webrtc/MediaEngineWebRTCAudio.cpp
--- a/dom/media/webrtc/MediaEngineWebRTCAudio.cpp	Thu Jan 27 22:29:35 2022 +0000
+++ b/dom/media/webrtc/MediaEngineWebRTCAudio.cpp	Thu Jan 27 22:29:35 2022 +0000
@@ -408,7 +408,7 @@
   MOZ_ASSERT(mState == kAllocated || mState == kStopped);
 
   // This check is unreliable due to potential in-flight device updates.
-  // Multiple input devices are reliably excluded in OpenAudioInputImpl(),
+  // Multiple input devices are reliably excluded in ConnectDeviceInputImpl(),
   // but the check here provides some error reporting most of the
   // time.
   CubebUtils::AudioDeviceID deviceID = mDeviceInfo->DeviceID();
@@ -428,7 +428,7 @@
 
         track->GraphImpl()->AppendMessage(MakeUnique<StartStopMessage>(
             track, inputProcessing, StartStopMessage::Start));
-        track->OpenAudioInput(deviceID, inputProcessing, principal);
+        track->ConnectDeviceInput(deviceID, inputProcessing, principal);
       }));
 
   ApplySettings(mCurrentPrefs);
@@ -460,7 +460,7 @@
         track->GraphImpl()->AppendMessage(MakeUnique<StartStopMessage>(
             track, inputProcessing, StartStopMessage::Stop));
         MOZ_ASSERT(track->DeviceId().value() == deviceInfo->DeviceID());
-        track->CloseAudioInput();
+        track->DisconnectDeviceInput();
       }));
 
   MOZ_ASSERT(mState == kStarted, "Should be started when stopping");
@@ -1171,7 +1171,7 @@
 
 void AudioInputTrack::Destroy() {
   MOZ_ASSERT(NS_IsMainThread());
-  CloseAudioInput();
+  DisconnectDeviceInput();
 
   MediaTrack::Destroy();
 }
@@ -1322,9 +1322,9 @@
   mInputProcessing = std::move(aInputProcessing);
 }
 
-nsresult AudioInputTrack::OpenAudioInput(CubebUtils::AudioDeviceID aId,
-                                         AudioDataListener* aListener,
-                                         const PrincipalHandle& aPrincipal) {
+nsresult AudioInputTrack::ConnectDeviceInput(
+    CubebUtils::AudioDeviceID aId, AudioDataListener* aListener,
+    const PrincipalHandle& aPrincipal) {
   MOZ_ASSERT(NS_IsMainThread());
   MOZ_ASSERT(GraphImpl());
   MOZ_ASSERT(!mInputListener);
@@ -1346,7 +1346,7 @@
   return NS_OK;
 }
 
-void AudioInputTrack::CloseAudioInput() {
+void AudioInputTrack::DisconnectDeviceInput() {
   MOZ_ASSERT(NS_IsMainThread());
   MOZ_ASSERT(GraphImpl());
   if (!mInputListener) {